api/tests/Behat/bootstrap/v2/AppNotification/AppNotificationTrait.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Behat\v2\AppNotification;
+
+trait AppNotificationTrait
+{
+    private string $validationMsg = 'Please enter a message';
+
+    /**
+     * @When I visit the service notification page
+     */
+    public function iNavigateToTheClientsSearchPage()
+    {
+        $this->iVisitTheNotificationPage();
+        $this->iAmOnAdminNotificationPage();
+    }
+
+    /**
+     * @When I set a service notification
+     */
+    public function iSetAServiceNotification()
+    {
+        $this->fillField('setting[content]', 'Lorem ipsum');
+        $this->selectOption('setting[enabled]', '1');
+        $this->pressButton('setting[save]');
+    }
+
+    /**
+     * @When I set a service notification and see it on the client login page
+     */
+    public function iSetAServiceNotificationAndSeeItOnTheClientPage()
+    {
+        $this->fillField('setting[content]', 'Lorem ipsum');
+        $this->selectOption('setting[enabled]', '1');
+        $this->pressButton('setting[save]');
+
+        $this->iVisitTheClientLoginPage();
+        $this->iAmOnClientLoginPage();
+
+        $notification = $this->getSession()->getPage()->find('css', '.behat-region-service-notification > .opg-alert > .behat-region-alert-message > p')->getText();
+        assert('Lorem ipsum' == $notification);
```

Comment:
As per others below - `AssertTrait::assertStringEqualsString()` would be good here for consistency.